src/trackinfoobject.h

```diff
@@ -378,7 +371,7 @@ class TrackInfoObject : public QObject {
     // Number of times the track has been played
     int m_iTimesPlayed;
     // Replay Gain volume
-    float m_fReplayGain;
+    double m_fReplayGain;
```

Comment:
We have no rule to for additional type specific prefixes, except p for pointers.
(I am not a big fan of it, since the IDEs are doing a good job showing the types.)
In this case "f" is misleading. Use d or nothing.